packages/contractkit/src/wrappers/Accounts.ts

```diff
@@ -69,44 +71,75 @@ export class AccountsWrapper extends BaseWrapper<Accounts> {
    * @param address The address of the account
    * @return Returns `true` if account exists. Returns `false` otherwise.
    */
-  isSigner: (address: string) => Promise<boolean> = proxyCall(this.contract.methods.isAuthorized)
+  isSigner: (address: string) => Promise<boolean> = proxyCall(
+    this.contract.methods.isAuthorizedSigner
+  )
 
   /**
    * Authorize an attestation signing key on behalf of this account to another address.
-   * @param account Address of the active account.
-   * @param attestationSigner The address of the signing key to authorize.
+   * @param signer The address of the signing key to authorize.
+   * @param proofOfSigningKeyPossession The account address signed by the signer address.
    * @return A CeloTransactionObject
    */
   async authorizeAttestationSigner(
-    account: Address,
-    attestationSigner: Address
+    signer: Address,
+    proofOfSigningKeyPossession: Signature
   ): Promise<CeloTransactionObject<void>> {
-    return this.authorizeSigner(SignerRole.Attestation, account, attestationSigner)
+    return toTransactionObject(
+      this.kit,
+      this.contract.methods.authorizeAttestationSigner(
+        signer,
+        proofOfSigningKeyPossession.v,
+        proofOfSigningKeyPossession.r,
+        proofOfSigningKeyPossession.s
+      )
+    )
   }
   /**
    * Authorizes an address to sign votes on behalf of the account.
-   * @param account Address of the active account.
-   * @param voteSigner The address of the vote signing key to authorize.
+   * @param signer The address of the vote signing key to authorize.
+   * @param proofOfSigningKeyPossession The account address signed by the signer address.
    * @return A CeloTransactionObject
    */
   async authorizeVoteSigner(
-    account: Address,
-    voteSigner: Address
+    signer: Address,
+    proofOfSigningKeyPossession: Signature
   ): Promise<CeloTransactionObject<void>> {
-    return this.authorizeSigner(SignerRole.Vote, account, voteSigner)
+    return toTransactionObject(
+      this.kit,
+      this.contract.methods.authorizeVoteSigner(
+        signer,
+        proofOfSigningKeyPossession.v,
+        proofOfSigningKeyPossession.r,
+        proofOfSigningKeyPossession.s
+      )
+    )
   }
 
   /**
    * Authorizes an address to sign consensus messages on behalf of the account.
-   * @param account Address of the active account.
-   * @param validationSigner The address of the signing key to authorize.
+   * @param signer The address of the signing key to authorize.
+   * @param proofOfSigningKeyPossession The account address signed by the signer address.
+   * @param proofOfBlsKeyPossession Proof-of-possession generated for the corresponding BLS key. Only needed if a validator has been registered.
```

Comment:
This is currently not implemented?

Reply:
Whoops! This was accidentally left over from the coupled-key implementation. BLS key rotation now happens in `Validators`